admin/app/tools/LoadBalancer.scala

```diff
@@ -88,20 +86,20 @@ object LoadBalancer extends GuLogging {
 
   def refresh(): Unit = {
     log.info("starting refresh LoadBalancer ELB DNS names")
-    credentials.foreach { _ =>
-      val client = ElasticLoadBalancingClient
-        .builder()
-        .credentialsProvider(AWSv2.credentials)
-        .region(Region.of(conf.Configuration.aws.region))
-        .build()
-      val response = client.describeLoadBalancers(DescribeLoadBalancersRequest.builder().build())
-      val elbs = response.loadBalancerDescriptions()
-      client.close()
-      val newLoadBalancers = loadBalancers.map { lb =>
-        lb.copy(url = elbs.asScala.find(_.loadBalancerName() == lb.id).map(_.dnsName()))
-      }
-      agent.send(newLoadBalancers)
+
+    val client = ElasticLoadBalancingClient
+      .builder()
+      .credentialsProvider(AWSv2.credentials)
+      .region(Region.of(conf.Configuration.aws.region))
+      .build()
+    val response = client.describeLoadBalancers(DescribeLoadBalancersRequest.builder().build())
+    val elbs = response.loadBalancerDescriptions()
+    client.close()
+    val newLoadBalancers = loadBalancers.map { lb =>
+      lb.copy(url = elbs.asScala.find(_.loadBalancerName() == lb.id).map(_.dnsName()))
     }
+    agent.send(newLoadBalancers)
```

Comment:
Tested this locally. The agent still gets populated with the loadbalancers info

<img width="211" height="360" alt="image" src="https://github.com/user-attachments/assets/11630aeb-51c0-458b-9f2d-d63ffc421130" />